pkgs/applications/misc/actiona/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  wrapQtAppsHook,
+
+  bluez,
+  libnotify,
+  libXdmcp,
+  libXtst,
+  opencv,
+  qtbase,
+  qtmultimedia,
+  qtscript,
+  qttools,
+  qtx11extras,
+  qtxmlpatterns,
+
+  textToSpeechSupport ? false,
+  qtspeech,
+}:
+
+let
+  # For some reason qtscript wants to use the same version of qtbase as itself
+  # This override makes it think that they are the same version
+  qtscript' = qtscript.overrideAttrs (oldAttrs: {
+    inherit (qtbase) version;
+    postPatch = ''
+      substituteInPlace .qmake.conf \
+          --replace-fail ${oldAttrs.version} ${qtbase.version}
+    '';
+  });
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "actiona";
+  version = "3.10.2";
+
+  src = fetchFromGitHub {
+    owner = "Jmgr";
+    repo = "actiona";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-4RKCNEniBBx0kDwdHVZOqXYeGCsH8g6SfVc8JdDV0hI=";
+    fetchSubmodules = true;
+  };
+
+  patches =
+    lib.singleton ./fix-paths.patch
```

Comment:
I notice you do not like `[]`.

Reply:
I would prefer [].

Reply:
The new formatter forces it to be on the same line, which is not the best IMO, maybe I can force it to be on multiple lines if I add a comment.

Reply:
The new formatter does not like 
```nix
patches = [
  ./fix-paths.patch
];
```
?

Where is this on https://github.com/NixOS/rfcs/pull/166 ?

Reply:
Well,  the the [Hammering Guides](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md) cited by @drupol pontificate the patches should be commented.

Reply:
I think the special thing about this formatter is that it will format to the same result given the same AST (+comments), so there is only one stable formatting.
And short 1 element arrays will be placed into a single line.

Reply:
Let's add a patch comment here.